### DISCUSSION TITLE
versioning issue when building a custom version

### DISCUSSION THREAD

**Discussion by sgrizzi** [iNavFlight/inav#9210] [Q&A]
I am trying to build a custom version derived from target BetaflightF4 in order to be able to add a BMI270 gyro/acc with SPI interface.
The development environment is in MacOS.
To start from a known working version, I first tried to compile the standard BetaflightF4 target available in inav-master.
I can successfully build a .hex file but I get it for version 7.0.0 (inav_7.0.0_BETAFLIGHTF4.hex).
My INAV configurator is 6.1. When I try and flash the local firmware I get the error that I need to upgrade first.

Can anybody help me find a way to build a version 6.1.1 -compatible hex code? Or point me where I can find the description on how to build a specific version?

**Comment by MrD-RC** [comment]
You need to base your branch in your fork on the 6.1.1 tag from the master origin.

**Reply by sgrizzi** [reply]
Perfect! Thanks a lot. I could not find the proper tag.
I will have a go with version 6.1.1 ...
